src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.ts

```diff
@@ -44,10 +47,8 @@ export class ContributionQuestionEditAnswerFormComponent
     return points;
   }
 
-  /**
-   * Opens a modal.
-   */
-  openModal(modal: any): void {
-    this.modalService.open(modal);
+  openHelpModal(): void {
```

Comment:
You can make `equalShareHelp` as a argument here so you don't need to declare `@ViewChild('equalShareHelp')`